src/main/java/net/kodehawa/mantarobot/commands/ActionCmds.java

```diff
@@ -338,129 +381,216 @@ public Cuddle() {
+                    "commands.action.stare", "stare", "commands.action.lonely.stare", "commands.action.self.stare", true
+            );
+        }
+    }
+
+    @Name("holdhands")
+    public static class HoldHandsText extends ImageActionCmd {
+        public HoldHandsText() {
+            super(
+                    "Hold Hands", "Hold someone's hands", EmoteReference.HEART,
+                    "commands.action.holdhands", "handholding", "commands.action.lonely.holdhands", "commands.action.self.holdhands", true
+            );
+        }
+    }
+
+    @Name("cuddle")
+    public static class CuddleText extends ImageActionCmd {
+        public CuddleText() {
+            super(
+                    "Cuddle", "Cuddles someone", EmoteReference.HEART,
+                    "commands.action.cuddle", "cuddle", "commands.action.lonely.cuddle", "commands.action.self.cuddle"
+            );
+        }
+    }
+
+    @Name("blush")
+    public static class BlushText extends ImageActionCmd {
+        public BlushText() {
+            super(
+                    "Blush", "Blushes at someone", EmoteReference.HEART,
+                    "commands.action.blush", "blush", "commands.action.lonely.blush", "commands.action.self.blush", true
+            );
+        }
+    }
+
+    @Name("nuzzle")
+    public static class NuzzleText extends ImageActionCmd {
+        public NuzzleText() {
+            super(
+                    "Nuzzle", "Nuzzles the specified user.", EmoteReference.TALKING,
+                    "commands.action.nuzzle", NUZZLE.get(), "commands.action.lonely.nuzzle", "commands.action.self.nuzzle", true
+            );
+        }
+    }
+
+    @Name("bloodsuck")
+    public static class BloodsuckText extends ImageActionCmd {
+        public BloodsuckText() {
+            super(
+                    "Bloodsuck", "Sucks the blood of a user", EmoteReference.TALKING,
+                    "commands.action.bloodsuck", BLOODSUCK.get(), "commands.action.lonely.bloodsuck", "commands.action.self.bloodsuck", true
+            );
+        }
+    }
+
+    public static class Tsundere extends TextActionCmd {
+        public Tsundere() {
+            super("Y-You baka!", EmoteReference.MEGA + "%s", TSUNDERE.get());
+        }
+    }
+
+    public static class Lewd extends ImageCmd {
+        public Lewd() {
+            super("T-Too lewd!", "lewd", "commands.action.lewd");
+        }
+    }
+
+    @Alias("mew")
+    public static class Meow extends ImageCmd {
+        public Meow() {
+            super("Meows at the specified user.", MEOW.get(), "commands.action.meow");
+        }
+
+    }
+
+    public static class Nom extends ImageCmd {
+        public Nom() {
+            super("*nom nom*", "nom", "commands.action.nom");
+        }
+    }
+
+    public static class Facedesk extends ImageCmd {
+        public Facedesk() {
+            super("When it's just too much to handle.", "banghead", "commands.action.facedesk", true);
+        }
```

Comment:
We should consider making a Hybrid class for these types of commands that handles both text and slash. If you look more closely you can see that each and every single one of these has the same exact class structure, they just rely on 2 different Contexts - which we can probably abstract away.